custom/ewsghana/models.py

```diff
@@ -173,7 +132,5 @@ def domain_pre_delete_receiver(domain, **kwargs):
     from corehq.apps.domain.deletion import ModelDeletion
     return [
         ModelDeletion('ewsghana', 'FacilityInCharge', 'location__domain'),
-        ModelDeletion('ewsghana', 'EWSExtension', 'domain'),
```

Comment:
@gcapalbo It's not removed. I only removed comma: https://github.com/dimagi/commcare-hq/pull/11042/files#diff-251f5e8a24177a37339ee2edcd513efbR135